src/main/java/ch/njol/skript/bukkitutil/EnchantmentUtils.java

```diff
@@ -40,4 +74,53 @@ public static Enchantment getByKey(String key) {
 		return Enchantment.getByKey(NamespacedKey.minecraft(key));
 	}
 
+	@SuppressWarnings("deprecation")
+	@Nullable
+	public static Enchantment parseEnchantment(String s) {
+		s = s.toLowerCase(Locale.ROOT);
+		// First try to parse from the lang file
+		Enchantment enchantment = PATTERNS.get(s);
+		if (enchantment != null)
+			return enchantment;
+
+		// If that fails, we move forward with getting from key
+		s = s.replace(" ", "_");
+		NamespacedKey key;
+		try {
+			if (s.contains(":")) {
+				key = NamespacedKey.fromString(s);
+			} else {
+				key = NamespacedKey.minecraft(s);
+			}
+		} catch (IllegalArgumentException ignore) {
+			return null;
+		}
+		if (key == null)
+			return null;
+		if (HAS_REGISTRY) // Registry added in Bukkit 1.14
+			return Registry.ENCHANTMENT.get(key);
+		return Enchantment.getByKey(key);
+	}
+
+	@SuppressWarnings("null")
+	public static Collection<String> getNames() {
+		return NAMES.values();
+	}
+
+	@SuppressWarnings("null")
+	public static String toString(final Enchantment enchantment) {
+		// If we have a name in the lang file, return that first
+		if (NAMES.containsKey(enchantment))
+			return NAMES.get(enchantment);
```

Comment:
might be better to just call `get` and see if it is null

i wonder if it's worth storing unknown names on the map though? then we could just do this all when the language is initialized